extensions/identity-hub/src/main/java/org/eclipse/dataspaceconnector/identityhub/models/RequestStatus.java

```diff
@@ -23,7 +25,7 @@ public class RequestStatus extends Status {
     public static final RequestStatus DID_NOT_FOUND = new RequestStatus(404, "Target DID not found within the Decentralized Web Node");
     public static final RequestStatus ERROR = new RequestStatus(500, "The request could not be processed correctly");
 
-    private RequestStatus(int code, String detail) {
+    private RequestStatus(@JsonProperty("code") int code, @JsonProperty("detail") String detail) {
```

Comment:
How about using Builder together with JsonDeserialize, to be consistent with the EDC codebase?

Reply:
I changed it
FYI, to do that I had to change the Status class to make code and detail protected and not final.
I also needed to change the field code to be an Integer instead of an int, to be able to create an instance of the RequestStatus int he builder and validate if the field was set or not.